examples/seismic/plotting.py

```diff
@@ -14,13 +14,15 @@
 
 def plot_perturbation(model, model1, colorbar=True):
     """
-    Plot a two-dimensional velocity difference from two seismic :class:`Model`
+    Plot a two-dimensional velocity perturbation from two seismic `Model`
     objects.
 
-    :param model: :class:`Model` object of first velocity model.
-    :param model1: :class:`Model` object of the second velocity model.
-    :param source: Coordinates of the source point.
-    :param receiver: Coordinates of the receiver points.
+    Parameters
+    ----------
+    model : Model
+        object of first velocity model.
```

Comment:
All descriptions start with a capital letter.

Also, this should simply be:
"The first velocity model."

Reply:
Fixed first and second